requirements.txt

```diff
@@ -1,4 +1,5 @@
 pip>=9.0.1
-pytest>=3.6,<7.0
+pytest>=3.6
 pytest-runner
 pytest-cov
+nvidia-ml-py>=12.0.0,<13.0.0a0
```

Comment:
As this is now in the `test` requirements in `pyproject.toml`, think we can drop this

Reply:
Is the suggestion to drop the `requirements.txt` file altogether? That seems fine to me. I only left it because it is used in the Dockerfile (though I don't personally know if anyone is using that).

Reply:
Update: I removed `requirements.txt` now that the Dockerfile doesn't need it.